rbx_binary/src/core.rs

```diff
@@ -91,6 +91,24 @@ pub trait RbxReadExt: Read {
         Ok(())
     }
 
+    fn read_interleaved_u32_array(&mut self, output: &mut [u32]) -> io::Result<()> {
+        let mut buffer = vec![0; output.len() * mem::size_of::<u32>()];
```

Comment:
`u32` has an explicitly defined size in Rust (4 bytes). Do we need to use `mem::size_of`?

Reply:
It makes it more explicit what the number is for, and it compiles down anyway.

Reply:
Forgot that it's determined at compilation. Oops! Nevermind!